middleware/middleware-1.3.11.md

```diff
@@ -0,0 +1,63 @@
+
+We especially focused on the stability of log exports and recurring operations like the self-tests and time updates, which should now also occur less often (and therefore reduce the number of consumed signatures).
+
+## Performance improvements for hardware TSEs
+While introducing the stability improvements mentioned above, we also worked hard to increase the performance of the **Swissbit, CryptoVision and Epson** TSEs. This is mostly reflecting in the performance of the log file exports, which - depending on the TSE - should now consume up to 50% less time.
+
+## New feature: Opt-out of telemetry collection in online mode
+The fiskaltrust Middleware collects usage and especially error telemetry, using Microsoft Application Insights. Previously, it was only possible to opt-out of the telemetry collection by using the Middleware in the offline mode, which comes with several disadvantages (e.g. no data upload). 
+
+We therefore introduced a new Launcher parameter that completely deactivates all logging to Application Insights, `--telemetry-optout`.
+
+Please keep in mind that using this will prevent us from actively reacting to issues in the respective customer installations, as we will be completely dependent on receiving inquiries and log files in those case.
+
+## Bug fix: Uploading data was sometimes failing for large queues
+We resolved an issue that lead to sometimes failing uploads to our cloud services for large queues, as the boundaries previously uploaded data was not always included properly. No data was lost due to this issue, and everything will be properly uploaded after a CashBox rebuild (which also updates the automatically added HelipadHelper).
+
+We also fixed a related issue that sometimes completely prevented uploading data when running on Linux.
+
+## Bug fix: The order of installation parameters is important
+We noticed and resolved an issue that lead to errors when using the fiskaltrust Launcher with specific orders of parameters.
+
+## Bug fix: Z_NUMBER is not always calculated correctly 
+We resolved an issue that could lead to wrong Z_NUMBERS if the `from` and `to` parameters where specified when generating DSFinV-K exports.
+
+## How to update
+Existing configurations with versions greater than 1.3.1 continue to work, **but we strongly recommend users of Hardware TSEs to update, as this new version should vastly improve the Middleware's reliability**.
+
+As always, updates can be rolled out by selecting the new version in the fiskaltrust.Portal and re-building the configuration. The updated Middleware instances will then automatically pull the new packages at the next startup.
+
+## Affected packages
+Packages not listed here were not updated, as we decided to not increase the version of unchanged packages. All packages with versions greater or equal to 1.3.1 are compatible with each other (it is e.g. possible to use _fiskaltrust.Middleware.SCU.Swissbit.1.3.1_ with the new queue packages).
```

Comment:
Didn´t we also release the client packages?

Reply:
Good catch! I added a section and the packages